source/_integrations/ld2410_ble.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: LD2410 BLE
+description: Instructions on how to integrate LD2410 BLE devices into Home Assistant.
+ha_category:
+  - Sensor
+  - Presence Detection
+ha_bluetooth: true
+ha_release: 2023.2
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@930913'
+ha_domain: ld2410_ble
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates LD2410B sensors from [Hi-Link](http://www.hlktech.net/) into Home Assistant using Bluetooth.
```

Comment:
Is there an example of where can I purchase this device?

Reply:
The main listing is https://www.aliexpress.com/item/1005004351593073.html, where there are a number of options - Bluetooth/non-Bluetooth, devkit/bare-board, pins/no pins. Obviously the non-Bluetooth variant won't work with this integration :wink:

Reply:
Maybe warning that there are multiple modules of this module and users should be sure to purchase the -B version as the others don't have bluetooth.

Although that listing implies the C does as well but it is aliexpress so....

Reply:
Purchase section added.